scripts/hotspot.py

```diff
@@ -162,12 +205,15 @@ def process_vcf(infile,outfile,genomic_hs,hugo_hs,fid,aa='Amino_acids',pos='Prot
     vcfin.header.info.add("HotSpotAllele",'A','Integer',"Is the corresponding ALT allele on a hotspot (0 for false; 1 for true).")
     vcfout = pysam.VariantFile(outfile,'w',header=vcfin.header)
     csq_items = vcfin.header.info["CSQ"].description.split(' Format: ')[1].split('|')
+    flagged = []
     total_hotspots_found = 0
     for rec in vcfin.fetch():
         genomic_hotspot_found = False
```

Comment:
Something is off about the genomic hotspot annotation. I used an input that I know has the TERT promoter hot spot, and it was not flagged. Input bed:
```
chr5	1295112	1295113	TERT
chr5	1295134	1295135	TERT
```
Input file: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/pbta-consensus-calls/files/5e27713ee4b09d9a44907df0/

Reply:
Yeah I was thrown off by the example genomic hotspots given: https://github.com/kgaonkar6/OpenPBTA-analysis/blob/combine-snv/analyses/hotspots-detection/input/tert_promoter_hotspots.tsv. The ones there have a 1-based start so I assumed it was an inclusive end position. I'd say it's appropriate to enforce BED format for this input as that is the most widely accepted way to collate genomic positions.